packages/eslint-plugin/tests/rules/member-ordering.test.ts

```diff
@@ -4471,7 +4517,7 @@ class Foo {
 abstract class Foo {
   abstract B: string;
   abstract A(): void;
-  public C(): {};
+  public C() {}
```

Comment:
This is probably a typo. The following TS error occurs because `{}` is the return type
`Function implementation is missing or not immediately following the declaration.`